### ISSUE TITLE
Legend title not announced by screen readers

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#14872]
#### Expected behaviour
When adding a title to the chart legend, this should be read by screen readers.

#### Actual behaviour
Legend titles are not exposed to assistive tech.

#### Live demo with steps to reproduce
https://jsfiddle.net/e32sfayo/

**Comment by oysteinmoseng** [comment]
Note: Should also add basic support for color axis announcements - e.g. axis range.
